### ISSUE TITLE
Detect Pebble template library

### ISSUE THREAD

**Issue by h3xstream** [find-sec-bugs/find-sec-bugs#521]
## Description

Pebble is unsafe if user input are given as part of the template. (Just like Freemarker and Velocity)

Ref: https://research.securitum.com/server-side-template-injection-on-the-example-of-pebble/

Similar detector : https://github.com/find-sec-bugs/find-sec-bugs/tree/master/findsecbugs-plugin/src/main/java/com/h3xstream/findsecbugs/template

## Code to detect

```java
PebbleEngine engine = new PebbleEngine.Builder().build();
PebbleTemplate compiledTemplate = engine.getLiteralTemplate(userInput);

Map<String, Object> context = new HashMap<>();
context.put("name", "Mitchell");

Writer writer = new StringWriter();
compiledTemplate.evaluate(writer, context);

String output = writer.toString();
```

**Comment by sa160690** [comment]
@h3xstream, can I do this?

**Comment by sa160690** [comment]
Hi @h3xstream, I have raised a pull request.

**Comment by h3xstream** [comment]
Implemented in #522
